test/spec/tokenManager.js

```diff
@@ -0,0 +1,189 @@
+define(function(require) {
+  var OktaAuth = require('OktaAuth');
+  var tokens = require('../util/tokens');
+  var util = require('../util/util');
+
+  function setupSync(options) {
+    options = options || {};
+    return new OktaAuth({
+      url: 'https://auth-js-test.okta.com',
+      tokenManager: {
+        storage: options.type
+      }
+    });
+  }
+
+  beforeEach(function() {
+    localStorage.clear();
+    sessionStorage.clear();
+  });
+
+  describe('general tokenManager', function() {
+    it('defaults to localStorage', function() {
+      var client = setupSync();
+      client.tokenManager.add('key', tokens.standardIdTokenParsed);
```

Comment:
nit: can we use a better key name here? i.e. 'test-idToken' or something like that